spec/views/posts/new.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "posts/new", type: :view do
+  before(:each) do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] RSpec/HookArgument: Omit the default :each argument for RSpec hooks.